src/pages/components/tabs-extended.mdx

```diff
@@ -16,107 +16,124 @@ import ResourceLinks from 'components/ResourceLinks';
 <AnchorLinks>
   <AnchorLink>Overview</AnchorLink>
   <AnchorLink>Tabs extended</AnchorLink>
-  <AnchorLink>Tabs extended media</AnchorLink>
+  <AnchorLink>Tabs extended with media</AnchorLink>
+  <AnchorLink>Behaviors</AnchorLink>
   <AnchorLink>Feedback</AnchorLink>
 </AnchorLinks>
 
-## Overview
+<InlineNotification>
 
-Tabs are useful for displaying parallel pieces of content that share the same context. It has the additional benefit of reducing screen height, but should be used carefully because some information is hidden on load.
+**Breaking change:** Previously, tabs extended was a separate component in Carbon for IBM.com v1. In v2, tabs extended will use Carbon Design System's [tab](https://carbondesignsystem.com/components/tabs/usage/) directly with automatic style updates.
 
-Our Tabs extended component references the Carbon Design System's [tabs component](https://carbondesignsystem.com/components/tabs/usage/), and it is built with additional features to optimize for editorial content.
+</InlineNotification>
 
-## Tabs extended
+## Overview
 
-On desktop, Tabs extended renders as tabs. On mobile, the component renders as an accordion, making it easy to read and access each section while scrolling.
+Tabs extended displays parallel pieces of related content, allowing users to explore the displayed content of their choice. It also has the benefit of reducing the amount of scrolling the user may have to do on a given page, but should be used carefully because some information is hidden on load. If the information in a tab would be key to the page content, consider laying out the content using a different component.
 
-Tab labels wrap up to two lines before being truncated. Truncated labels are fully displayed in tooltips on hover and focus.
+The tabs extended component uses the Carbon Design System's [tabs](https://carbondesignsystem.com/components/tabs/usage/) component directly, and includes automatic style updates to optimize for editorial content.
 
 <Row>
 <Column colMd={8} colLg={8}>
```

Comment:
This image looks a bit too small to really showcase the component. I'd suggest showing it at 12 columns or reducing the padding. 
I've been creating the original content at 100%, then scaling by 62% per the image production guidelines and seeing which size frame it fits on best, our 12 column components are really unfortunate on all the sizes tbh.. 608 has no extra space and 928 is too much space.

Reply:
Looks like we're missing the blue selected indicator on all the images